ticket-app/src/main/java/com/chudzik/services/ValidationService.java

```diff
@@ -0,0 +1,86 @@
+package com.chudzik.services;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.springframework.stereotype.Service;
+
+import com.chudzik.exceptions.InvalidScreeningException;
+import com.chudzik.exceptions.SeatReservationException;
+import com.chudzik.models.Screening;
+import com.chudzik.models.Seat;
+
+@Service
+public class ValidationService {
+
+    private ValidationService() {
+
+    }
+
+    public static boolean validateName(String name) {
+        if(name == null || name.length()> 50)
+        {
+            return false;
+        }
+        Pattern namePattern = Pattern.compile("[A-Z][a-z]{2,}");
```

Comment:
Czy ten i regex od nazwiska poprawnie ogarnie polskie imiona/nazwiska? :detective:

Reply:
Tak, przyjmując że locale jest ustawione na pl_PL. Żeby to nie było koniecznym warunkiem należy zmienić pattern na `"(?U)\\p{Upper}\\p{Lower}{2,}"` i analogicznie dla surname pattern.